src/asmcnc/apps/drywall_cutter_app/screen_config_filechooser.py

```diff
@@ -403,6 +403,10 @@ def refresh_filechooser(self):
             if self.filechooser.selection[0] != 'C':
                 if not DWTConfig.is_valid_configuration(self.filechooser.selection[0]):
                     DWTConfig.fix_config(self.filechooser.selection[0])
+                if float(DWTConfig.get_config_version(self.filechooser.selection[0])) > float(DWTConfig.get_latest_config_version()):
```

Comment:
This crashes right away, because you can't fix a config that is too new. I assume you only changed the version from "2.0" to "3.0"
But a new version will have new or changed fields in the json. When I added this to my "too new"-config it triggers the except BEFORE showing the popup:

"some_new_element": {
		"value_1"= 1,
		"value_2"= "test",
		"value_3"= true
	},
	
First check the version and show the popup, then call fix_config